controllers/team_membersController.js

```diff
@@ -0,0 +1,88 @@
+import team_members_get from "../models/team_membersModel.js";
+export async function adduser(req, res) {
+    try {
+        const data = req.body;
+        console.log("New team member data: ", data);
+
+        if (!data || Object.keys(data).length === 0) {
+            return res.status(400).json({
+                success: false,
+                message: "Request body is empty or invalid"
+            });
+        }
+
+        const requiredFields = ["name", "role", "email", "description"];
+        for (const field of requiredFields) {
+            if (!data[field]) {
+                return res.status(400).json({
+                    success: false,
+                    message: `${field} is required`
+                });
+            }
+        }
+
+        const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+        if (!emailRegex.test(data.email)) {
+            return res.status(400).json({
+                success: false,
+                message: "Invalid email format"
+            });
+        }
+        const existingMember = await team_members_get.findOne({ email: data.email });
+        if (existingMember) {
+            return res.status(409).json({
+                success: false,
+                message: "A team member with this email already exists"
+            });
+        }
+
+        const new_team_member = new team_members_get(data);
+        await new_team_member.save();
+
+        res.status(201).json({ success: true, new_team_member });
```

Comment:
Why are we not sending a message here, once a new user is created, why do we need to send **new_team_member**, frontend doesn't need this info, send a success and message